rastervision2/core/data/vector_source/vector_source_config.py

```diff
@@ -11,7 +11,7 @@ class VectorSourceConfig(Config):
         ('The default class_id to use if class cannot be inferred using other '
          'mechanisms. If a feature defaults to a class_id of None, then that feature '
          'will be deleted.'))
-    class_id_to_filter: Optional[Dict[int, Optional[Dict]]] = Field(
```

Comment:
I think this can contain nested lists so it should just be `Sequence`, not `Sequence[str]`

Reply:
I looked into this, and it doesn't appear to work using a nested sequence.  It makes some sense that it wouldn't since it's not clear if multiple tests would be processed as an 'any of' or 'all of' relationship.  I'm having trouble finding examples of Mapbox styles that illustrate how this would work, so any pointers would be useful.

Should we proceed with single tests or actually update the code to do the right thing with lists of lists (assuming an _all of_ approach for lists of qualifications)?

Reply:
I found the docs for the filter language: https://docs.mapbox.com/mapbox-gl-js/style-spec/other/#other-filter (please update the description string while you're at it) From this it's apparent that lists of lists are possible. I don't think we need to update anything in the RV code for this to work. This is where it's implemented: https://github.com/azavea/raster-vision/blob/master/rastervision/data/vector_source/label_maker/filter.py

Reply:
Aha.  Makes sense.  Fixed now.